editoast/src/config.rs

```diff
@@ -0,0 +1,29 @@
+use std::collections::HashMap;
```

Comment:
I would create a `search` module at the root of editoast with a submodule named `config` (this file)